package.json

```diff
@@ -5,29 +5,83 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "vue-tsc && vite build",
-    "preview": "vite preview",
-    "fmt": "yarn prettier --write ."
+    "build": "tsc && vite build",
+    "build-contracts": "node ./scripts/build-contracts.js",
+    "deploy-contracts": "node ./scripts/deploy.js",
+    "evolve-contracts": "node ./scripts/deploy.js --evolve",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
   },
   "dependencies": {
-    "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
-    "arweave": "^1.13.7",
-    "bulma": "^0.9.4",
+    "@headlessui/react": "^1.7.16",
+    "@hookform/resolvers": "^3.2.0",
+    "@isomorphic-git/lightning-fs": "^4.6.0",
+    "@uiw/codemirror-extensions-langs": "^4.21.11",
+    "@uiw/codemirror-theme-github": "^4.21.11",
+    "@uiw/react-codemirror": "^4.21.11",
+    "@uiw/react-md-editor": "^3.23.5",
+    "arweave": "^1.14.4",
```

Comment:
Would ideally migrate to ArweaveKit in the future

Reply:
Yes. I've created a task for it just now. https://communitylabs.atlassian.net/browse/PL-118?atlOrigin=eyJpIjoiMzNjNGMzNTY3ZTBiNGI4YjhjOTBkZTI3MGE0MzAyODgiLCJwIjoiaiJ9